core/src/main/java/com/scalar/db/api/OperationBuilder.java

```diff
@@ -432,6 +433,16 @@ interface WhereOr<T> {
     T where(ScanBuilder.AndConditionSet conditions);
   }
 
+  interface WhereConditions<T> {
+    /**
+     * Appends the specified conditions.
+     *
+     * @param conditions an or-wise set of and-wise conditions.
+     * @return the operation builder
+     */
+    T where(Set<Set<ConditionalExpression>> conditions);
```

Comment:
Can you please provide some examples to help us understand when this would be useful? I'm curious as to why the existing methods don't suffice.

Reply:
@brfrn169 Thank you for the question. When we have a set of conjunctions, we need to call `where()` and `or()` several times and change the behavior depending on the number of conjunctions. I just wanted to avoid managing an intermediate builder like the following.

```java
BuildableScanAll buildableScanAll =
    Scan.newBuilder()
        .namespace(statement.namespaceName)
        .table(statement.tableName)
        .all();

BuildableScanAllWithOngoingWhereOr ongoing = null;
for (Conjunction conjunction : conjunctions) {
  // snip conversion of conjunction in the SQL layer -> conditional expression
  if (ongoing == null) {
     ongoing = builder.where(ConditionSetBuilder.andConditionSet(conditions).build());
  } else {
     ongoing.or(ConditionSetBuilder.andConditionSet(conditions).build());
  }
}
```

Reply:
@brfrn169 That's the point that bothered me (since it's somewhat imbalanced in and/or set, how about adding a message like "internal purpose", and so on and so on...). Anyway, after reconsidering, I think your idea is better. Fixed in acd3be2. Thank you!